src/terminal/terminal-handlers.c

```diff
@@ -885,6 +885,10 @@ int guac_terminal_csi(guac_terminal* term, unsigned char c) {
                 if (flag != NULL)
                     *flag = true;
 
+                /* Open alternate screen buffer */
+                if (argv[0] == 1049)
```

Comment:
Might be worth `#define` a constant for this value...

Reply:
It's done!